File: zar.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>
int main()
{
srand(time(NULL));
int x = rand() % 6 + 1;
int y = rand() % 6 + 1;

if(x == 6 && y == 5)
        printf("Dau cu zaru 6 5 n-am nevoie de servici\n");
else if(x == 6 && y == 6)
        printf("Dau cu zaru 6 6 ca mine nu se mai naste\n");
else
        printf("Dau cu zaru %d %d\n", x, y);

return 0;
}

File: zar.py
#!/usr/bin/env python3
import random

x = random.randint(1, 6)
y = random.randint(1, 6)

if x == 6 and y == 5:
        print("Dau cu zaru 6 5 n-am nevoie de servici")
elif x == 6 and y == 6:
        print("Dau cu zaru 6 6 ca mine nu se mai naste")
else:
        print(f"Dau cu zaru {x} {y}")

